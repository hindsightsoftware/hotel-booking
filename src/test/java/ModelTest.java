import org.approvaltests.Approvals;
import org.junit.Test;

import java.util.Date;

public class ModelTest {

    @Test
    public void testModel(){
        Date checkin = new Date();
        checkin.setTime(1514764800);
        Date checkout = new Date();
        checkout.setTime(1514851200);

        Booking booking = new Booking("Mark", "Winteringham", 111, true, checkin, checkout, "Breakfast");

        Approvals.verify(booking.toString());
    }

}
