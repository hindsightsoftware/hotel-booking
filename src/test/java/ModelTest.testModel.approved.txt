Booking{firstname='Mark'
, lastname='Winteringham'
, price=111
, booked=true
, bookingDates=
BookingDates{
checkin=Sun Jan 18 13:46:04 GMT 1970
, checkout=Sun Jan 18 13:47:31 GMT 1970}
, additional='Breakfast'
}
